lib/date_time/date_time.c

```diff
@@ -9,32 +9,39 @@
 #include <zephyr/types.h>
 #include <device.h>
 #include <stdio.h>
-#include <stdlib.h>
+#include <posix/time.h>
+
 #if defined(CONFIG_DATE_TIME_MODEM)
 #include <nrf_modem_at.h>
+#if defined(CONFIG_DATE_TIME_AUTO_UPDATE)
```

Comment:
call for at_monitor_init() is flagged behind different flags.

And thus there is a compiler warning if compiling without CONFIG_DATE_TIME_AUTO_UPDATE

Reply:
Fixed these. Will show up in the next update.